client/src/app/site/motions/services/motion-block-repository.service.ts

```diff
@@ -112,4 +112,14 @@ export class MotionBlockRepositoryService extends BaseRepository<ViewMotionBlock
             .getViewModelListObservable()
             .pipe(map(viewMotions => viewMotions.filter(viewMotion => viewMotion.motion_block_id === block.id)));
     }
+
+    /**
+     * Retrieves motion block(s) by name
+     * TODO: check if a title is unique for a motionBlock
+     * @param title Strign to check for
+     */
+    public getMotionBlockByTitle(title: string): MotionBlock {
+        const filteredBlocks = this.DS.find(MotionBlock, block => block.title === title);
```

Comment:
Just a return here should be enough. If the DS cannot find anything, this should be null.